docs/data-sources/add-configurations.md

```diff
@@ -1,29 +1,27 @@
 ---
 layout: default
-title: Add data sources
+title: Add configurations
 nav_order: 3
-parent: Connecting to data sources
+parent: Data Sources
 ---
 
-# How to analyze data from software development repositories
+# Add configurations
```

Comment:
Maybe this can be elaborated upon. It's not going to be added to the slug so a bit more descriptive might be good.

Reply:
Ok